jenkins-plugin/pom.xml

```diff
@@ -320,6 +335,157 @@
         <artifactId>ssh-credentials</artifactId>
         <version>1.13</version>
       </dependency>
+      <dependency>
```

Comment:
@abayer can you please tell us why you need to add 20+ dependencies including things that seem to be completely unrelated to the purpose of this plugin such as `commons-fileupload` or `ant`

Reply:
First, I added them to `<dependencyManagement>` - i.e., it just says "if a dependency of ours depends on this artifact, use this version". Second, it was needed to get the upper bounds enforcer checks that were added to the plugin parent to pass. @jglick can explain the rationale for that change, but the gist is that you already depended on all these artifacts (and many, many, many more - 339 total by my count!) via transitive dependencies, mainly from `maven-plugin` and `maven-invoker-plugin` as well as core, but the versions were inconsistent and needed to be rationalized.

Reply:
I looked too quickly at the diff, seem to be in the `<dependencyManagement>` section, not in the `<dependencies>`section. I need to double check

Reply:
Thanks for the explanation @abayer, sorry for the time you spent in this explanation, I wrote my comment "I need to double check" at the same time you were writing your explanation.